imaginedsf-custom-theme/acf-json/group_5d5c783dc10e5.json

```diff
@@ -0,0 +1,182 @@
+{
+    "key": "group_5d5c783dc10e5",
+    "title": "Proposals",
```

Comment:
For the Proposals field group, in the inner repeater of maps, we only want to have the year and the map, no title necessary (it's already saved in the Map).  For the map, you'll want to use a different field type: instead of a text field with a link to the map, there's a field type called Post Object and you can set it to filter to only Map post types that will let the user search for a Map specifically.